src/components/application_manager/src/commands/hmi/navi_is_ready_response.cc

```diff
@@ -48,6 +48,12 @@ void NaviIsReadyResponse::Run() {
   bool is_available = false;
   if (object[strings::msg_params].keyExists(strings::available)) {
     is_available = object[strings::msg_params][strings::available].asBool();
+    const HmiInterfaces::InterfaceState interface_state =
+        is_available ? HmiInterfaces::STATE_AVAILABLE
+                     : HmiInterfaces::STATE_NOT_AVAILABLE;
+    HmiInterfaces& hmi_interfaces = application_manager_.hmi_interfaces();
+    hmi_interfaces.SetInterfaceState(HmiInterfaces::HMI_INTERFACE_Navigation,
+                                     interface_state);
```

Comment:
@dtrunov , Is it possible to create method with this changes in base class? I see 3 places with same code